### ISSUE TITLE
Writing of a long string times out when using TLSv1_1 method

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8693]
|[<img alt="anchat1990's avatar" src="https://avatars.githubusercontent.com/u/7135503?s=50" width="50" height="50">](https://github.com/anchat1990)| @anchat1990 reported|
|-|-|
|Trac ID|trac#8693|
|Type|defect|
|Created|2016-07-27 00:04:32Z|

Writing of a very long string fails after timeout when using the TLSv1_1 method from the OpenSSL library. This is fixed when the buffersize is reduced from 2^16 to 2^14 bytes in the write function of TLSMemoryBIOProtocol in tls.py

LINK TO GITHUB PULL REQUEST: https://github.com/twisted/twisted/pull/424

<details><summary>Searchable metadata</summary>

```
trac-id__8693 8693
type__defect defect
reporter__anchat1990 anchat1990
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469577872119637 1469577872119637
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 26b6d334f3eb57bff5f64a3180855c200d9c2223
```
#!CommitTicketReference repository="" revision="26b6d334f3eb57bff5f64a3180855c200d9c2223"
Merge pull request #430 from twisted/8693-anchat1990-twisted-tls-hugeWrite-error-1

Author: anchat1990
Reviewer: rodrigc
Fixes: #8693

Updated the tests to reflect that hugeWrite fails when using TLSv1_1
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
